Gemfile.lock

```diff
@@ -1,14 +1,14 @@
 PATH
```

Comment:
The presence of Gemfile.lock theoretically should not break anything but it does nothing for gem users and so is usually not supposed to be checked in. You might want to exclude it from `spec.files` in gemspec.